### DISCUSSION TITLE
Error message:  Pressure to PQ_flash

### DISCUSSION THREAD

**Discussion by nschnait** [oemof/tespy#449] [Q&A] [answered]
Hello community, 

i am new to the TESPy package and am currently trying out many examples to increase my understanding. 
Several times I had the problem that the following error message appears when using the refrigerant R290:
 
ValueError: Pressure to PQ_flash [ 5e+06 Pa] must be in range [0.000171848 Pa, 4.2512e+06 Pa].

This error message also appears specifically in the example: 
First steps // 5 minutes after TESPy // Mini example // Modelling a heat exchanger

Link: https://oemof.github.io/heat-pump-tutorial/getting-started/5-minutes-into-tespy.html

Can someone please suggest a solution to this problem? Thank you very much!

**Comment by fwitte** [comment]
Hi,

thank you for reporting, does it happen with the code as is or if modified by you?

Thanks! Best

Francesco

**Reply by nschnait** [reply]
Hallo Francesco, 

vielen Dank für die schnelle Rückmeldung. Genau das passiert wenn der Code so ist. Zumindest bei diesem genannten Beispiel (siehe Foto). Bei einem von mir geänderten Beispiel hatte ich das Problem auch. Bin noch kein Profi in CoolProp - kann das Problem da liegen? 

Viele Grüße 

Nicolas 

![Pressure to PQ_flash](https://github.com/oemof/tespy/assets/152208079/552fe05c-8fb2-4f81-adeb-49791dcba0c6)

**Reply by nschnait** [reply]
Ich habe das Kältemittel R290 durch NH3 ersetzt das funktioniert.

**Reply by fwitte** [reply, marked as answer]
Du verwendest noch die alte API. Man sollte (bis auf in wenigen Nischenanwendungen) nicht mehr `fluid={"a": 1, "b":0}` sondern nur noch `fluid={"a": 1}` verwenden. Es kann sein, dass das der Grund für den Fehler ist.

In deinem environment musst du zum Aktualisieren von TESPy

`pip install tespy --upgrade` oder `conda update tespy` (je nachdem, ob du mit pip oder mit conda installiert hast)

ausführen und dann spyder neu starten.

Viele Grüße

**Reply by fwitte** [reply]
P.S.: Das Beispiel auf der gh Page werde ich eben aktualisieren.

**Reply by fwitte** [reply]
P.P.S.: Die Tutorial-Seite sollte in ca. 5 Minuten dann auch aktualisiert sein. Einfach nochmal neu laden, evtl. Cache leeren. Danke für die Rückmeldung :)

**Reply by nschnait** [reply]
Super, funktioniert, vielen Dank für die Unterstützung!
